### ISSUE TITLE
A real preview to render the final results?

### ISSUE THREAD

**Issue by gengshenghong** [jaredly/hexo-admin#22]
This is a nice plugin for posting blog articles. But I've some concerns. At the beginning, I am using some markdown editor to write articles, and find it is not good because there are some extended markdown syntax or other plugins (example: mathjax, diagram) which my editor may not support.

But, when I tried hexo-admin, it also cannot support these plugin, and I can confirm that if I open the post article in browser directly, it works (it shows my math equations, my diagrams). Can the "preview" of hexo-admin preview the actual final results (same with what you see in browser directly)? Or I've missed anything? If not supported, I do not see a reason to use hexo-admin instead of other markdown editors. Any idea?

Thanks,
Shenghong

**Comment by jaredly** [comment]
yeah, ideally I do want to support full rendering, i just haven't done the
work to get it in. I don't think it would be particularly hard

**Comment by gengshenghong** [comment]
@jaredly ,

Thank you for your confirmation. Looking forward to seeing the feature asap.
That should be a really important feature for hexo-admin (this is the first feature I was expecting when I see the plugin introduction, as I find it is difficult/impossible to find a markdown editor support all the plugins's syntax I want to use in hexo)

Thanks,
Shenghong

**Comment by scientificRat** [comment]
Any updates on this issue? This feature is still missing in the latest release

**Comment by daniel18qzx** [comment]
Any updates on this issue? This feature is still missing in the latest release
